pkg/storage/unified/resource/document.go

```diff
@@ -255,6 +259,7 @@ const SEARCH_FIELD_TITLE_SORT = "title_sort"
 const SEARCH_FIELD_DESCRIPTION = "description"
 const SEARCH_FIELD_TAGS = "tags"
 const SEARCH_FIELD_LABELS = "labels" // All labels, not a specific one
+const SEARCH_FIELD_FIELDS = "fields" // Index all fields
```

Comment:
every other value with the prefix "SEARCH_FIELD_" is the name of a searchable field -- this looks like it is a parameter?

Reply:
Ya probably not the right place for it. I'm going to get rid of this, and hardcode it in `bleve_mappings.go` where its used.